bin/qserv-test-integration.py

```diff
@@ -49,30 +49,52 @@
 from lsst.qserv.admin import commons
 from lsst.qserv.admin import logger
 from lsst.qserv.tests import benchmark
-from lsst.qserv.tests.unittest.testIntegration import suite
+from lsst.qserv.tests.unittest import testIntegration, testCall
 
 # ---------------------------------
 # Local non-exported definitions --
 # ---------------------------------
 _LOG = logging.getLogger()
 
 
+# To add test suites: add a name (key) and a factory lambda function (value) to the all_tests dictionary.
+# Then it will be available in the arguments to the script, and will be run by default if test names aren't
+# explicitly passed in by the caller.
+all_tests = {'testCall': lambda : testCall.suite(),
+             'testIntegration': lambda: testIntegration.suite(multi_node)}
+
+
 def _parse_args():
 
     parser = argparse.ArgumentParser(
-        description='''Qserv integration tests suite. Relies on python unit
-testing framework, provide test meta-data which can be used for example in a
-continuous integration framework or by a cluster management tool. Configuration values
-are read from ~/.lsst/qserv.conf.''',
+        description=("Qserv integration tests suite. Relies on python unit testing framework, provide test "
+                     "meta-data which can be used for example in a continuous integration framework or by a "
+                     "cluster management tool. Configuration values are read from ~/.lsst/qserv.conf."),
         formatter_class=argparse.ArgumentDefaultsHelpFormatter
     )
 
+    parser.add_argument('-t', '--run-tests',
+        help="The name of one or more test suites to execute. May include one or more of {}".format(
+             list(all_tests.keys())),
+        choices = all_tests.keys(),
+        nargs = '+',
+        default = all_tests.keys(),
```

Comment:
Do you want a `list()` around this and couple of lines above?

Reply:
no; IIRC `keys` returns an iterator object` and prints that into the string and in that case I want the values - a regular list to print in the regular list way. It seems the pythonic way to go from that iter to a list is to pass it to the `list` function as above. For `default`, using the iter returned by `keys` is fine.

Reply:
Well, `argparse` API does not explicitly say that passing an iterator to `choices` or `default` is OK (though it says that `choices` supports container types). I thought it would be safer to use actual container type here rather tan rely on undocumented features.

Reply:
And I'm probably confused here too. `keys()` does not return in iterator in Python3, it returns "KeysView" object which is probably good enough model for a container. Please ignore my cat's uninformed opinion 🐱

Reply:
Oh, right - KeysView. Not an iterator, but an iterable object.